pkg/health/client/modules_test.go

```diff
@@ -83,13 +81,21 @@ func (m *testMHappy) GetHealth(params *daemon.GetHealthParams, opts ...daemon.Cl
 					LastUpdated: "2s",
 				},
 				{
-					ModuleID:    "m2",
-					Level:       string(cell.StatusDegraded),
-					Message:     fmt.Sprintf("{\"name\": \"m2\", \"level\":\"Degraded\",\"message\":\"doh\", \"timestamp\": %q}", t2),
+					ModuleID: "a.b.c",
+					Level:    string(cell.StatusDegraded),
+					Message: fmt.Sprintf(`
+					{"level": "OK", "name": "a.b.c", "count": 1, "sub_statuses": [
```

Comment:
Could we just use `*UpdateStatus` and then marshal to make it more clear where this json comes from for someone maintaining this?